volatility3/framework/symbols/windows/extensions/crash.py

```diff
@@ -0,0 +1,27 @@
+# This file is Copyright 2019 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
Interesting that crash.py's year got updated, but this one got a 2019 year instead...  5;P  I think once they're in there it's no so bad, but they should probably go in with the right year at least...  5;)

Reply:
Fixed by 7b73f8d